app/views/new_document/choose_supertype.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_tag(choose_document_type_path, method: :get, enforce_utf8: false) do %>
+    <%= form_tag(choose_document_type_path, method: :get, enforce_utf8: false, data: { gtm: "new-document" } ) do %>
```

Comment:
Is this meant to be the same tag as choosing document type? and should they both reflect document type / supertype rather than new-document?

Reply:
Yes, I went through it with Andy